<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViaLactea</class>
 <widget class="QMainWindow" name="ViaLactea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>921</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vialactea - Visual Analytics client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWebEngineView" name="webView" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>0</height>
        </size>
       </property>
       <property name="url" stdset="0">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Load data</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="openLoadDataPushButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../visivo.qrc">
                 <normaloff>:/icons/FILE_OPEN.bmp</normaloff>:/icons/FILE_OPEN.bmp</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>3D</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="select3dPushButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string>Select</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="surveySelectorGroupBox">
          <property name="title">
           <string>Survey Selector</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="2">
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>GLIMPSE/MIPSGAL</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QCheckBox" name="glimpse_24_checkBox">
                  <property name="text">
                   <string>24 μm</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="glimpse_8_checkBox">
                  <property name="text">
                   <string>8.0 μm</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="glimpse_58_checkBox">
                  <property name="text">
                   <string>5.8 μm</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="glimpse_45_checkBox">
                  <property name="text">
                   <string>4.5 μm</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="glimpse_36checkBox">
                  <property name="text">
                   <string>3.6 μm</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QGroupBox" name="groupBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="title">
                <string>Hi-GAL</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QCheckBox" name="PLW_checkBox_2">
                  <property name="text">
                   <string>500 μm</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="PMW_checkBox">
                  <property name="text">
                   <string>350 μm </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="PSW_checkBox_12">
                  <property name="text">
                   <string>250 μm </string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="blue_checkBox_2">
                  <property name="text">
                   <string>70 μm </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="red_checkBox">
                  <property name="text">
                   <string>160 μm </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QGroupBox" name="groupBox_6">
               <property name="title">
                <string>WISE</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QCheckBox" name="wise_22_checkBox">
                  <property name="text">
                   <string>22 μm</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="wise_12_checkBox">
                  <property name="text">
                   <string>12 μm</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="wise_46_checkBox">
                  <property name="text">
                   <string>4.6 μm</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="wise_34_checkBox">
                  <property name="text">
                   <string>3.4 μm</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBox_7">
               <property name="title">
                <string>ATLASGAL</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QCheckBox" name="atlasgal_870_checkBox">
                  <property name="text">
                   <string>870 μm</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QGroupBox" name="groupBox_8">
               <property name="title">
                <string>BOLOCAM GPS</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QCheckBox" name="bolocam_11_checkBox">
                  <property name="text">
                   <string>1.1 mm</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QGroupBox" name="groupBox_9">
               <property name="title">
                <string>CORNISH</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <widget class="QCheckBox" name="cornish_6_checkBox">
                  <property name="text">
                   <string>5 GHz</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="selectionGroupBox">
          <property name="title">
           <string>Selection</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="noneRadioButton">
             <property name="text">
              <string>None</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="pointRadioButton">
             <property name="text">
              <string>Point</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rectRadioButton">
             <property name="text">
              <string>Rectangular</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="coordinateGroupBox">
          <property name="title">
           <string>Coordinates (center of selection)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>glon</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="glonLineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>glat</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="glatLineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="1">
              <widget class="QLabel" name="dlLabel">
               <property name="text">
                <string>dl</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>radius</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLineEdit" name="radiumLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>10</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="dbLabel">
               <property name="text">
                <string>db</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="dlLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>10</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="dbLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>10</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="saveToDiskCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Save to disk</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="fileNameLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectFsPushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../visivo.qrc">
              <normaloff>:/icons/FILE_OPEN.bmp</normaloff>:/icons/FILE_OPEN.bmp</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="queryPushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Query</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadTableButton">
          <property name="text">
           <string>Request mcutouts</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1266</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_session"/>
    <addaction name="actionLoadArepo"/>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
    <addaction name="actionLoad_SED_2"/>
   </widget>
   <widget class="QMenu" name="menuAiuto">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuVO">
    <property name="title">
     <string>VO</string>
    </property>
    <addaction name="actionConeSearch"/>
    <addaction name="actionHiPS2FITS"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVO"/>
   <addaction name="menuAiuto"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLoad_SED">
   <property name="text">
    <string>Load SED</string>
   </property>
  </action>
  <action name="actionLoad_SED_2">
   <property name="text">
    <string>Load SED</string>
   </property>
  </action>
  <action name="actionLoad_session">
   <property name="text">
    <string>Load session</string>
   </property>
  </action>
  <action name="actionConeSearch">
   <property name="text">
    <string>Cone Search</string>
   </property>
  </action>
  <action name="actionHiPS2FITS">
   <property name="text">
    <string>HiPS2FITS</string>
   </property>
  </action>
  <action name="actionLoadArepo">
   <property name="text">
    <string>Load AREPO Simulation...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../visivo.qrc"/>
 </resources>
 <connections/>
</ui>
